.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/ably-common"]
```

Comment:
I think we need some instructions in place so folks know to run git submodules (and install `chromium`) before running this test locally

Reply:
Thanks, have added a section to `CONTRIBUTING.md`.